portfolio/src/test/java/com/google/sps/CalendarGroupTest.java

```diff
@@ -0,0 +1,328 @@
+package com.google.sps;
+
+import com.google.sps.data.CalendarEvent;
+import com.google.sps.data.CalendarGroup;
+import com.google.sps.data.TimeRange;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CalendarGroupTest {
+
+  /** Start time of the scheduling block must not be later then end time. */
+  @Test(expected = IllegalArgumentException.class)
+  public void startTimeAfterEndTime() {
+    Instant startTime = Instant.now();
+    Instant endTimeBefore = startTime.minusSeconds(1000);
+    Instant endTimeAfter = startTime.plusSeconds(1000);
+
+    List<CalendarEvent> events = new ArrayList<CalendarEvent>();
```

Comment:
We can simplify these lines with,

List<CalendarEvent> events = ImmutableList.of(
    new CalendarEvent(...));

Reply:
Updated.